cpp/CMakeLists.txt

```diff
@@ -465,7 +465,16 @@ add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/include/types.h.jit
                    DEPENDS stringify
                    MAIN_DEPENDENCY ${CMAKE_CURRENT_SOURCE_DIR}/include/cudf/types.h)
 
-add_custom_target(stringify_run DEPENDS ${CMAKE_BINARY_DIR}/include/types.h.jit)
+add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/include/types.hpp.jit
+                   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include
+                   COMMAND ${CMAKE_BINARY_DIR}/stringify cudf/types.hpp > ${CMAKE_BINARY_DIR}/include/types.hpp.jit
```

Comment:
You can add multiple commands to this by adding more lines starting with `COMMAND`. So you can merge the stringification of `types.h` and `types.hpp`.

Reply:
Wouldn't it be better to keep them separate since each each command has a different dependency? Maybe these blocks could be simplified with a CMake macro though